.github/workflows/run-test-cases.yml

```diff
@@ -133,16 +133,16 @@ jobs:
             version: v1.21.5+k3s1
           - runtime: K3s-1.22
             version: v1.22.6+k3s1
-          - runtime: Kubernetes-1.17
```

Comment:
do we want to try to support the same versions across k8s, k3s, microk8s?

Reply:
Agreed, I'll update those now as well. 👍 

K8s failed to start for 1.22 - 1.25 so I'm checking to see what config broke there.

Reply:
Added support for all versions, note MicroK8s 1.25 had an issue with loading services. I'm opening a separate issue to investigate this since it seems to be affecting different services included with the microk8s (mostly kubelet from the snap pkg).

Other than that, all other versions are working